backend/Dockerfile

```diff
@@ -86,12 +92,12 @@ FROM base as production
 # Copy "binary"-files from build image
 COPY --from=build ${DOCKER_WORKDIR}/dist ./dist
 COPY --from=build ${DOCKER_WORKDIR}/node_modules ./node_modules
```

Comment:
No copy here of the node modules

Reply:
Agree!

Reply:
Is not working @Mogge . See other cases.